skaffold.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: skaffold/v2alpha1
+apiVersion: skaffold/v2beta9
```

Comment:
is this a downgrade?  I'm really unfamiliar with the skaffold api versions

Reply:
nope, upgrade so that the skaffold.yaml matches the the current skaffold documentation